### PR TITLE
Improve desktop page-based navigation.

### LINKED ISSUE #88: Move footer buttons to the app header for larger devices.
They're too hard to find on tablet portrait and above devices. Affects #82

### PR DESCRIPTION
**Pull request by nylira**
I've implemented a new NiPageHeader/NiToolBar design that:

1. Works for mobile and desktop
2. Has hover labels for desktop
3. Has much better positioning on desktop

screen shot 2017-11-23 at 2 03 01 pm

I've also added a help button to the top right of the screen, because we need something like that. It opens to a modal for users to get help on our Matrix, Forum, or GitHub (for now).

screen shot 2017-11-23 at 2 02 54 pm

Fixes #88

### CODE REVIEW

**Comment by jolesbi** [comment]
amazing! well done!

**Comment by jolesbi** [comment]
i find the placement of the 'copy' button a bit confusing. what exactly is being copied if you click it? would it be better directly next to the copyable value?

**Comment by nylira** [comment]
Yeah that is a good point. We should put the copy button next to the address.

File: app/src/renderer/components/staking/PageNominate.vue [hunk 1]
```diff
@@ -122,6 +124,7 @@ import FormGroup from '../common/NiFormGroup'
 import FormMsg from '../common/NiFormMsg'
 import FormStruct from '../common/NiFormStruct'
```
Comment (jolesbi) [inline app/src/renderer/components/staking/PageNominate.vue]: don't need the .. anymore

Comment (nylira) [reply app/src/renderer/components/staking/PageNominate.vue]: Yep, will work on a PR to remove all of those relative links (soon).

**Pull request merged**